src/mainboard/clevo/adl-p/Kconfig

```
@@ -0,0 +1,68 @@
+if BOARD_SYSTEM76_NS70PU
```

```suggestion
if BOARD_CLEVO_NS70PU
```

along with all other instances